src/screens/GroupInfo/GroupMedia.js

```diff
@@ -2,11 +2,11 @@ import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs
 import * as React from 'react';
 import {SafeAreaView, StatusBar, StyleSheet, View, TouchableOpacity} from 'react-native';
 import Animated from 'react-native-reanimated';
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import Header from './elements/Header';
 import Link from './elements/Link';
 import Media from './elements/Media';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the `utils/theme` module is a good change. It's better to use a constant object for colors rather than individual color variables. This makes it easier to manage and update color schemes in the future.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->